packages/venia-ui/lib/components/RichContent/PageBuilder/utils.js

```diff
@@ -0,0 +1,162 @@
+/**
+ * Retrieve background images from a master format node
+ *
+ * @param node
+ * @returns {{mobileImage: null, desktopImage: null}}
+ */
+export function getBackgroundImages(node) {
+    const images = node.getAttribute('data-background-images');
+    const response = {
+        desktopImage: null,
+        mobileImage: null,
+        backgroundSize: node.style.backgroundSize,
+        backgroundPosition: node.style.backgroundPosition,
+        backgroundAttachment: node.style.backgroundAttachment,
+        backgroundRepeat: node.style.backgroundRepeat !== 'no-repeat'
+    };
+
+    if (images) {
+        const imagesStructure = JSON.parse(images.replace(/\\"/g, '"'));
+        if (imagesStructure.desktop_image) {
+            response.desktopImage = imagesStructure.desktop_image;
+        }
+        if (imagesStructure.mobile_image) {
+            response.mobileImage = imagesStructure.mobile_image;
+        }
+    }
+
+    return response;
+}
+
+const alignmentToFlex = {
+    top: 'flex-start',
+    middle: 'center',
+    bottom: 'flex-end'
+};
```

Comment:
You should make another one of these for the other axis.

Reply:
This currently just maps to the functionality we have in Page Builder, this'll be improved if needed within that features story at a later date.